onnxruntime/core/providers/cpu/quantization/qlinearconv.cc

```diff
@@ -664,10 +665,16 @@ Status QLinearConv<ActType>::Compute(OpKernelContext* context) const {
       memset(col_data, 0, SafeInt<size_t>(sizeof(ActType)) * group_col_buffer_size);
     }
   }
+
+  bool parallel_batch = is_symmetric_conv_ && channels_last_;
+
   if (use_indirection_buffer) {
     // Allocate indirection buffer pointers and prepare a padding vector for
     // the im2col transform.
-    auto* indirection_data = alloc->Alloc(SafeInt<size_t>(sizeof(const ActType*)) * kernel_size * output_image_size);
+    ind_buf_length = SafeInt<size_t>(kernel_size) * output_image_size;
```

Comment:
make it: ind_buf_length = SafeInt<size_t>(sizeof(const ActType*)) * kernel_size * output_image_size. and you don't need another SafteInt in alloc->Alloc